src/microsoft/office/license/user/microsoft-office-license-user.controller.js

```diff
@@ -35,25 +35,17 @@ angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicens
                     this.loaders.userPoll[item] = true;
 
                     this.license.pollUserDetails(this.$scope.currentLicense, item, this.$scope)
-                        .then(() => {
-                            this.loaders.userPoll.item = false;
-                            return this.delayedGetUser();
-                        })
-                        .catch((err) => {
-                            this.loaders.userPoll.item = false;
-                            return err;
-                        });
+                        .then(() => this.delayedGetUser())
+                        .catch((err) => err)
+                        .finally(() => { this.loaders.userPoll.item = false; });
                 }
                 return details;
             })
             .catch((err) => {
                 this.loaders.userPoll.item = false;
                 return err;
-            });
-    }
-
-    onTransformItemDone () {
-        this.loaders.users = false;
+            })
+            .finally(() => { this.loaders.users = false; });
```

Comment:
Loader logic was removed from html by your modifications. You could remove it from the controller as well.

Reply:
Yes removed.